dali/pipeline/operator/arg_helper_test.cc

```diff
@@ -83,17 +83,17 @@ TEST(ArgValue, TensorInput_3D) {
 
 TEST(ArgValueTests, Constant_0D) {
   int nsamples = 5;
-  auto spec = OpSpec("opname").AddArg("argname", 0.123f);
-  ArgValue<float, 0> arg("argname", spec);
+  auto spec = OpSpec("Erase").AddArg("shape", 0.123f);
```

Comment:
Note: Using TryGetArgument* required that the spec points to an existing operator.